gen3/bin/kube-setup-secrets.sh

```diff
@@ -206,36 +206,48 @@ if [[ -f "$(gen3_secrets_folder)/creds.json" ]]; then # update fence secrets
   if ! g3kubectl get secrets/fence-config > /dev/null 2>&1; then
     # load updated fence-config.yaml into secret if it exists
     fence_config=$(gen3_secrets_folder)/apis_configs/fence-config.yaml
-    if [[ -f ${fence_config} ]]; then
-      echo "loading fence config from file..."
-      if g3kubectl get secrets/fence-config > /dev/null 2>&1; then
-        g3kubectl delete secret fence-config
-      fi
+    fence_pub_config=$(gen3_secrets_folder)/apis_configs/fence-config-public.yaml
```

Comment:
The public fence-config.yaml should come from the manifest folder - something like `$(g3k_manifest_folder)/fence/fence-config.yaml`